models/cartDao.js

```diff
@@ -0,0 +1,88 @@
+const appDataSource = require("./dataSource");
+
+const createCart = async (productId, quantity, userId) => {
+  const result = await appDataSource.query(
+    `insert into cart (products_id , quantity , users_id) values(?,?,?);`,
+    [productId, quantity, userId]
+  );
+
+  return result;
+};
+
```

Comment:
```suggestion
const createCart = async (productId, quantity, userId) => {
  const result = await appDataSource.query(`
    INSERT INTO cart (
      products_id , quantity , users_id
    ) VALUES (?, ? , ?);`,
    [productId, quantity, userId]
  );

  return result;
};

```

SQL 구문에 대한 가독성을 높이기 위해 대문자를 추천드립니다. 다른 함수에도 적용해 주세요 :)